evap/evaluation/models.py

```diff
@@ -1668,7 +1670,48 @@ def create_superuser(self, *, email, password=None, first_name=None, last_name=N
         return user
 
 
-class UserProfile(AbstractBaseUser, PermissionsMixin):
+class EvapBaseUser(models.Model):
+    """This is strongly related to the django.contrib.auth.base_user.AbstractBaseUser model, but does not define natural_key."""
+
+    USERNAME_FIELD: str
+
+    # set unusable password by default: We don't want password resets.
+    password = models.CharField(_("password"), max_length=128, default=partial(make_password, None))
```

Comment:
We don't need a `password_changed` event in our code, and we also do not use [password validation](https://docs.djangoproject.com/en/5.1/topics/auth/passwords/#module-django.contrib.auth.password_validation).

We could add an assertion somewhere that would trigger if anyone ever tried to configure password validation, something like
```python3
assert AUTH_PASSWORD_VALIDATORS == [], "Password validation configured, but evap will not apply it"
```
that crashes at application startup.

Reply:
just to mention here: the deprecation is because of the `*args` parameter (https://github.com/django/django/commit/3915d4c70d0d7673abe675525b58117a5099afd3), not because of the `password_changed` call.

However, that assertion seems still valuable to have :+1: